ouroboros-consensus/test-consensus/Test/Dynamic/LeaderSchedule.hs

```diff
@@ -40,6 +40,7 @@ tests = testGroup "Dynamic chain generation"
       , praosSlotsPerEpoch = 3
       , praosLeaderF       = 0.5
       , praosLifetimeKES   = 1000000
+      , praosSlotLength    = slotLengthFromSec 2
```

Comment:
Was this supposed to be 2 or 20? Just checking.

Reply:
2, as the slot length of Praos will also be 2s, right?